Stratis.Bitcoin.Features.RPC/RpcSettings.cs

```diff
@@ -58,58 +58,69 @@ public void Load(NodeSettings nodeSettings)
             var config = nodeSettings.ConfigReader;
 
             this.Server = config.GetOrDefault<bool>("server", false);
-            if (!this.Server)
-                return;
+            this.RPCPort = nodeSettings.Network.RPCPort;
+
+            if (this.Server)
+            {
+                this.RpcUser = config.GetOrDefault<string>("rpcuser", null);
+                this.RpcPassword = config.GetOrDefault<string>("rpcpassword", null);
+
+                var defaultPort = config.GetOrDefault<int>("rpcport", nodeSettings.Network.RPCPort);
```

Comment:
You can remove the variable 'defaultPort' here.

Reply:
Done